ui/app/src/state/epics/pathNavigator.ts

```diff
@@ -399,7 +399,10 @@ export default [
             actions.push(pathNavigatorRefresh({ id: navigator.id }));
           } else if (!navigator.isRootPathMissing && navigator.currentPath.startsWith(sourcePath)) {
             actions.push(pathNavigatorSetCurrentPath({ id: navigator.id, path: navigator.rootPath }));
-          } else if (navigator.currentPath === parentOfTargetPath || navigator.currentPath === parentOfSourcePath) {
+          } else if (
+            withoutIndex(navigator.currentPath) === parentOfTargetPath ||
+            withoutIndex(navigator.currentPath) === parentOfSourcePath
```

Comment:
What's the logic?

Would you need to equalise the `parentOf*` to also strip index for this comparison. Make sure you're not looking at the issue through a straw.

Reply:
`parentOf*` is retrieved by using `getParentPath` that already uses `withoutIndex`. A refresh of the pathNavigator is done when the currentPath is the same as the parent of the target or source. The issue was that the current path was matching one of the parents, but not entiretly due to the /index.xml being present in the currentPath